js/web/docs/webgpu-operators.md

```diff
@@ -0,0 +1,54 @@
+## Operators Support Table
+
+The following table shows ONNX
+operators from which onnx opset version are currently supported in WebGPU EP by ONNX Runtime Web. For example,
```

Comment:
How about?
"The following table shows ONNX operators and the supported opset-versions in WebGPU EP by ONNX Runtime Web"